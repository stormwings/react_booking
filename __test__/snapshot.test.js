import { getCharacter } from "../snapshot";
import rick from "../rick.json";

describe("Es hora de las intantaneas", () => {
  // la primera vez se va a generar un folder con los snapshot
  // las siguientes solo será una comprobación
  test("Snapshot", () => {
    expect(getCharacter(rick)).toMatchSnapshot();
  });

  test("Tenemos una excepsion dentro del codigo", () => {
    const user = {
      createAt: new Date(),
      id: Math.floor(Math.random() * 20),
      name: "Oscar Barajas Tavares"
    };
    expect(user).toMatchSnapshot({
      id: expect.any(Number),
      createAt: expect.any(Date)
    });
  });
});
