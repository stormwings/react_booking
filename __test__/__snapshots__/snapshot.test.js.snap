// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Es hora de las intantaneas Snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Human",
  "status": "Alive",
}
`;

exports[`Es hora de las intantaneas Tenemos una excepsion dentro del codigo 1`] = `
Object {
  "createAt": Any<Date>,
  "id": Any<Number>,
  "name": "Oscar Barajas Tavares",
}
`;
